articles/iot-edge/production-checklist.md

```diff
@@ -316,6 +316,20 @@ You can do so in the **createOptions** of each module. For example:
 
 For the most efficient IoT Edge deployment scenario, consider integrating your production deployment into your testing and CI/CD pipelines. Azure IoT Edge supports multiple CI/CD platforms, including Azure DevOps. For more information, see [Continuous integration and continuous deployment to Azure IoT Edge](how-to-continuous-integration-continuous-deployment.md).
 
+
+### Each IoT Edge device should have its own unique device connection string 
+
+Please make sure unique device connection string is **not** inadvertently **shared** between 2 or more separate IoT Edge devices.
```

Comment:
We use bold formatting to reference terminology or UI labels, not for emphasis. Also, I think you can simplify this sentence to something like:

`A device connection string cannot be shared between two or more IoT Edge devices.`